Frontend/ui-customer/src/Services/Backend.purs

```diff
@@ -722,14 +722,18 @@ type Markers = {
 }
 
 driverTracking :: String -> Markers
-driverTracking _ = {
-    srcMarker : if isPreviousVersion (getValueToLocalStore VERSION_NAME) (getPreviousVersion "") then "ic_auto_map" else "ic_vehicle_nav_on_map",
+driverTracking variant = {
```

Comment:
@MercyQueen Can we depreciate getPreviousVersion check if all the user are migrated